resources/js/Pages/Articles/Components/WaveSurferPlayer.tsx

```diff
@@ -3,9 +3,11 @@ import { useTranslation } from "react-i18next";
 import WaveSurfer, { type WaveSurferOptions } from "wavesurfer.js";
 import { Button } from "@/Components/Buttons";
 import { Icon } from "@/Components/Icon";
+import { useDarkModeContext } from "@/Contexts/DarkModeContex";
```

Comment:
heads up for both you and me whatever PR is merged first, this or https://github.com/ArdentHQ/dashbrd/pull/332, this is a typo and my dark mode PR fixes that to be `DarkModeContext` (missing `t` at the end). whatever PR is merged first, will need to fix this